app/src/lib/stores/github-user-store.ts

```diff
@@ -126,13 +141,23 @@ export class GitHubUserStore extends BaseStore {
    */
   public async getMentionableUsersMatching(
     repository: GitHubRepository,
-    text: string,
-    maxHits: number = 100
+    query: string,
+    maxHits: number = 5
   ): Promise<ReadonlyArray<IMentionableUser>> {
-    const users = await this.getMentionableUsers(repository)
+    assertPersisted(repository, this.getMentionableUsers.name)
```

Comment:
sigh.. and here I thought I was being clever by referencing the name in a way that would survive refactors and then I fell for the old copy paste mistake. Great catch. If only we could get https://docs.microsoft.com/en-gb/dotnet/api/system.runtime.compilerservices.callermembernameattribute?view=netframework-4.7.1 in TS/JS

Reply:
When I saw this PR it reminded me to our old friend [`arguments.callee`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments/callee) 😅 

We could achieve something similar by throwing an Error and parsing its stacktrace (that's super slow but since it should rarely happen - and when it does we already throw an error - it's not a super-bad-idea™️ #whatcouldgowrong).

Reply:
Uh now that I think about this, depending on our uglify configuration the function name may get minimized during our production build process, and `this.getMentionableUsersMatching.name` could return garbage (I'd be surprised if we used the advanced minification that modifies class methods though).

Reply:
> Uh now that I think about this, depending on our uglify configuration the function name may get minimized during our production build process, and `this.getMentionableUsersMatching.name` could return garbage (I'd be surprised if we used the advanced minification that modifies class methods though).

Dang it, I didn't even think of that. I'm gonna stop trying to be clever, it's just not for me. I removed the method name entirely in https://github.com/desktop/desktop/pull/10121/commits/a405a93a4042611efb5dbca84c331972e5a84674. If we have one it'll be part of the stack trace anyway.